### DISCUSSION TITLE
Discord v13 release

### DISCUSSION THREAD

**Discussion by vdemcak** [discordjs/discord.js#6175] [Q&A] [answered]
Hey!

Do you have any planned release dates for the v13 or are there any dev/beta branches? 
Thanks!

**Comment by almostSouji** [comment, marked as answer]
`npm i discord.js@dev` is released every 12h with the current state of #master, please see the pins in themed support channels on discord.gg/djs for guide previews.

We are trying to prevent quick follow-up releases, so we're waiting on some API finalizations on Discord's end before releasing v13 to stable.

**Reply by vdemcak** [reply]
Thanks!
